esp/src/src/FileSpray.ts

```diff
@@ -107,14 +107,15 @@ var LandingZonesFilterStore = declare([ESPRequest.Store], {
         }
     },
     preProcessRow: function (row) {
-        var fullPath = this.dropZone.machine.Directory + row.name;
+        var fullPath = this.dropZone.machine.Directory  + "/" + (row.Path===null ? "" : (row.Path + "/"));
         lang.mixin(row, {
             NetAddress: this.dropZone.machine.Netaddress,
             Directory: this.dropZone.machine.Directory,
             calculatedID: this.dropZone.machine.Netaddress + fullPath,
+            OS: this.dropZone.machine.OS,
             fullPath: fullPath,
-            fullFolderPath: row.Path,
```

Comment:
fullPath no longer needed?

Reply:
well fullPath should be removed and replaced with fullFolderPath as it is using the new logic, but it would need to be cleaned form other areas of the code so that we reference fullFolderPath instead of fullPath